jxl/srgb-ref.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<title>JPEG XL test reference</title>
+
+<body>
+<p class="desc">Test pixel values of a JPEG XL file</p>
+<p class="output">Actual output:</p>
```

Comment:
I would suggest omitting this text here and in the test. When looking at the reference (this file) it's really the expected output.

Reply:
make sense, done